packages/manager/apps/pci-instances/src/utils/operation/operations.utils.ts

```diff
@@ -0,0 +1,25 @@
+import { TOperation } from '@/types/operation/entity.type';
```

Comment:
You can move these utility functions close to the related hook / component because it only targets the operations logic.

Reply:
yay I had trouble decided where to put this file